larpandoracontent/LArControlFlow/MasterAlgorithm.cc

```diff
@@ -34,6 +34,7 @@ namespace lar_content
 
 MasterAlgorithm::MasterAlgorithm() :
     m_workerInstancesInitialized(false),
+    m_larCaloHitVersion(1),
```

Comment:
Is `1` the current LArCaloHit then? Just need to settle on whether that was 0 or 1, I suppose. Matters most in that it sets a precedent for future such changes.

Reply:
I think for versioning my preference is for the first "released" version to be version 1, rather than 0, though no strong opinion

Reply:
I don't have a strong feeling on this; we could maybe add a line of comment to explain the convention, whichever it is we settle on.

Reply:
I don't have a strong feeling on this; we could maybe add a line of comment to explain the convention, whichever it is we settle on.

Reply:
I also don't mind. I guess I had an expectation that our existing unversioned LArCaloHit might be henceforth referred to as the original/zeroth version, but it could equally just be the "first" version. Let's stick with original being v1, and new (first addition of daughter volume id) being v2, then, as implemented?

Reply:
Sounds good to me, thanks!